login.txt

```diff
@@ -0,0 +1 @@
+login info
```

Comment:
Use camel casing here for better readability

Reply:
Added camel casing in letters